lib/axp209.js

```diff
@@ -15,12 +15,16 @@ function AXP209(bus, address) {
   this._i2c = new I2C(bus, address);
 
   this._reads = {};
+  
 }
 
 util.inherits(AXP209, events.EventEmitter);
 
 AXP209.prototype.open = function() {
   this._i2c.open();
+  
+  this._configureVcc18Adc();
+  //this._resetVcc18();
```

Comment:
I think these should go in `AXP209.prototype.pinMode`, so they are configured when someone sets the pin mode to analog.